forms/listbox.cpp

```diff
@@ -205,6 +205,24 @@ void ListBox::update()
 	}
 	if (scroller)
 	{
+		size_t mainAxisSize = 0;
+		switch (ListOrientation)
+		{
+			case Orientation::Vertical:
+				mainAxisSize = Size.y;
+				break;
+			case Orientation::Horizontal:
+				mainAxisSize = Size.x;
+				break;
+			default:
+				// shouldn't happen
+				return;
+		}
+		size_t mainAxisContentSize =
+		    ItemSize * Controls.size() + ItemSpacing * (Controls.size() - 1);
```

Comment:
Could this go negative and blow up if Controls.size() is zero? As size_t is unsigned it'll be MAX_SIZE_T?